drl.go

```diff
@@ -85,12 +81,11 @@ func (d *DRL) uniqueID(s Server) string {
 }
 
 func (d *DRL) Close() {
-	if d.IsOpen() {
-		d.stopC <- struct{}{}
+	d.once.Do(func() {
```

Comment:
Can refactor this without sync.once, with a d.open.Swap, like in cache. Leaving a breadcrumb for later.